### PR TITLE
FAQ Update Nov 4.21

### PR DESCRIPTION
**Pull request by ninosamson**
# Fixes or Implements AB#11448

-   [x] Enhancement
-   [ ] Bug
-   [ ] Documentation

## Description
Update FAQ

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [x] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES | NO

### Browsers Tested

-   [ ] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/views/faq.vue [hunk 1]
```diff
@@ -98,7 +88,7 @@
                                 </li>
                             </ul>
                             <p>
-                                Future data sets will include health authority
+                                In future, Health Gateway will include health authority
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/faq.vue:91]: Should be "In the future".

**Review by pagustinqt20 — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/faq.vue [hunk 2]
```diff
@@ -270,12 +246,12 @@
...
-                                that immunizations administered by a pharmacy
-                                may be displayed in your profile as a dispensed
-                                medication rather than an immunization , but
-                                these will be added to the Registry in the
-                                future.
+                                clinics are not recorded in the Registry. 
+                                Immunizations administered by a pharmacy
+                                may be displayed as a dispensed
+                                medication rather than an immunization. 
+                                If you can’t find an immunization record,
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/faq.vue:253]: Do we have a policy about using or not using curly apostrophes and quotes?

**Review by MikeLyttle — CHANGES_REQUESTED** [review]
Needs linting.

File: Apps/WebClient/src/ClientApp/src/views/faq.vue (hunk 2 above)
Comment (ninosamson) [reply Apps/WebClient/src/ClientApp/src/views/faq.vue]: @pagustinqt20 @BrianMaki @sslaws For discussion.

File: Apps/WebClient/src/ClientApp/src/views/faq.vue (hunk 1 above)
Comment (ninosamson) [reply Apps/WebClient/src/ClientApp/src/views/faq.vue]: updated.

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**